include/battle_dome_cards.h

```diff
@@ -0,0 +1,13 @@
+#ifndef GUARD_BATTLE_DOME_CARDS_H
```

Comment:
Are these functions called from any other file but battle_dome_cards.c? If not, no need to add a header file.

Reply:
These functions are called in other components yet to be decompiled.  The idea is to have the header file now because it will be needed eventually.